tests/blueprints/test_plates.py

```diff
@@ -6,23 +6,17 @@
 
 def test_post_plates_endpoint_successful(app, client, samples, mocked_responses, mlwh_lh_samples):
     with patch(
-        "lighthouse.blueprints.plates.add_cog_barcodes",
-        return_value="TS1",
+        "lighthouse.blueprints.plates.add_cog_barcodes", return_value="TS1",
     ):
         ss_url = f"http://{app.config['SS_HOST']}/api/v2/heron/plates"
 
         body = json.dumps({"barcode": "123"})
         mocked_responses.add(
-            responses.POST,
-            ss_url,
-            body=body,
-            status=HTTPStatus.OK,
+            responses.POST, ss_url, body=body, status=HTTPStatus.OK,
         )
```

Comment:
These don't seem to match up with the way the linter has been handling things on my system. If I checkout dev and run `black .` as suggested in the Readme, it rolls-this change back.

Reply:
As discussed, will change this as a follow up release. We figured out it's because my `black` version was out of date.

Reply:
Andrew did this one in our bug fix release (https://github.com/sanger/lighthouse/pull/172)